app/models/order.rb

```diff
@@ -27,7 +27,7 @@ def sanitize_price(price)
   def add_items(cart)
     cart.contents.each do |item_id, quantity|
       item = Item.find(item_id)
-      quantity.times {items << item}
+      quantity.to_i.times {items << item}
```

Comment:
Hmm. @limsammy Shouldn't the quantity (the value of the hash) always be an integer already?